setup/quickstart/halyard.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: single
-title:  "Quick Full-Installations Using Halyard"
+title:  "Try out full installations using Halyard"
```

Comment:
These are not "full installations" anymore? I'm assuming this would be the single page that clicking on "Quickstarts" would take you to..?

Reply:
> This way we can use some of this content for re-doing the non-quickstart setup docs

Ah, OK, thanks for explaining. Given that part of the point of this restructuring is to give people _fewer_ options for potentially doing "full installations" of Spinnaker, do you think it would make sense to add a header to each one of these to explain that the **recommended** way of doing a full Spinnaker installation is to follow the "normal" setup guide?

Reply:
Yep, that seems reasonable; done.

Reply:
> Yep, that seems reasonable; done.

Thanks! Curious if we have any standard way of marking particular sections as deprecated/no longer recommended, but we can always take care of that in a follow-up.

/cc @dorbin

Reply:
I think our standard way is to delete and redirect.